cartridges/int_adyen_overlay/cartridge/scripts/adyenGetOriginKey.ds

```diff
@@ -79,7 +80,16 @@ function getOriginKey(args) {
     return responseObj;
 }
 
+function getOriginKeyFromRequest(protocol, host) {
+	var origin = protocol + "://" + host;
```

Comment:
We had this approach in the Magento2 plugin but then we had to realise that on the same host we can have test and live shops as well (i guess this is the case in SFCC as well) so what we did there was using the store id as a key instead of the url to store the origin key.